### PR TITLE
Feature: shutter animation

### PR DESCRIPTION
**Pull request by andi34**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/andi34/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [x] New feature

#### What changes did you make? (Give an overview)

Add a simple shutter animation while taking a picture. Implementation from @up-87,
fixed animation on retry / next collage image.
Made shutter animation an option inside Adminpanel, animation is enabled by default.

### CODE REVIEW

File: src/js/core.js [hunk 1]
```diff
@@ -341,6 +341,7 @@ const photoBooth = (function () {
     };
 
     api.cheese = function (photoStyle, retry = 0) {
+        api.stopPreviewVideo(CameraDisplayMode.COUNTDOWN);
         photoboothTools.console.logDev('Photostyle: ' + photoStyle);
```
Comment (andi34) [inline src/js/core.js]: ?

Comment (up-87) [reply src/js/core.js]: As I said I tried a bit around with it and tried to keep the cheese (Wouldn’t make any sense though. The photo was taken at the time of the fade to white) That‘s why I considered the branch dirty. This line can be deleted.

File: src/js/core.js [hunk 2]
```diff
@@ -426,6 +427,10 @@ const photoBooth = (function () {
     };
 
     api.callTakePicApi = function (data, retry = 0) {
+        const blocker = $('#blocker');
```
Comment (up-87) [inline src/js/core.js]: Should be moved up to the other jquery constants.

File: src/js/core.js [hunk 3]
```diff
@@ -434,6 +439,27 @@ const photoBooth = (function () {
                 totalTime = endTime - startTime;
                 photoboothTools.console.log('took ' + data.style, result);
                 photoboothTools.console.logDev('Taking picture took ' + totalTime + 'ms');
+                const aperture = $('#aperture');
```
Comment (up-87) [inline src/js/core.js]: Should be moved up to the other jquery constants.

**Comment by andi34** [comment]
@up-87 anything left? IMO can be rebased and merged if there's currently no further improvement.

**Pull request merged**